src/main/xar-resources/data/jmx/jmx.xml

```diff
@@ -14,42 +14,29 @@
-        available services in a standardized way. </para>
-    <para>The standard Java installation includes a simple client, JConsole, which will also display
-        the eXist-specific services. eXist also provides a command-line client for quick access to
-        server statistics and other information.</para>
-    <para>Right now, eXist only exposes a limited set of read-only services. Most of them are useful
-        for debugging purposes only.</para>
+    <para>eXist-db provides access to various management interfaces via Java Management Extensions (JMX). An agent in the Java virtual machine exposes agent services as <emphasis>MBeans</emphasis> that belong to different components running within the virtual machine. A JMX-compliant management application can then connect to the agent through the MBeans and access the available services in a standardized way. </para>
+    <para>The standard Java installation includes a simple client, JConsole, which will display the eXist-specific services. eXist also provides a command-line client for quick access to server statistics and other information.</para>
+    <para>eXist only exposes a limited set of read-only services. Most of them are useful for debugging purposes only.</para>
 
     <!-- ================================================================== -->
 
     <sect1 xml:id="enable">
         <title>Enabling the JMX agent</title>
 
-        <para>To enable the platform server within the host virtual machine, pass the following Java
-            system properties:</para>
+        <para>To enable the platform server on the host virtual machine, pass the following Java system properties:</para>
         <programlisting xlink:href="listings/listing-1.txt"/>
         <warning>
-            <para>These options makes the server publicly accessible. Please check the Oracle <link
-                xlink:href="https://docs.oracle.com/javase/1.5.0/docs/guide/management/agent.html">
-                JMX documentation</link> for details.</para>
+            <para>These options make the server publicly accessible. Please check the Oracle <link xlink:href="https://docs.oracle.com/javase/1.5.0/docs/guide/management/agent.html"> JMX documentation</link> for details.</para>
         </warning>
-        <para>The extension can now be activated by passing a <code>-j</code> or <code>-jmx</code>
-            command-line parameter to the eXist start scripts (<literal>client.sh</literal>,
-            <literal>startup.sh</literal> etc.). This parameter must be followed by the port number
-            through which the JMX/RMI connections are enabled. For instance:</para>
+        <para>The extension can now be activated by passing a <code>-j</code> or <code>-jmx</code> command-line parameter to the eXist start scripts (<literal>client.sh</literal>, <literal>startup.sh</literal> etc.). This parameter must be followed by the port number through which the JMX/RMI connections are enabled, for instance:</para>
```

Comment:
I don't think this is the case anymore - at least I can't seem to get `-j` to work with any of the `.sh` scripts. Have you tested this?

In additional it looks like there is now a new script called `jmxclient.sh` - what is that for and what does it do? We should probably add that to this documentation.

Reply:
Agree documentation probably needs updating and testing, but this section is unrelated to the section that this PR updates (which I did test, but didn't require any special parameters on start-up).

Suggest a new issue?

Reply:
I think we should try and cover it in this PR please.

Reply:
This should now be resolved.